src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,94 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+  justify-content: center;
+  padding-top: 50px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
```

Comment:
a lower cards row shouldn't have margin-bottom
use :nth-last-child(-n + 4) to select 4 last elements

Reply:
done